examples/tv-app/linux/include/tv-channel/TvChannelManager.cpp

```diff
@@ -51,6 +51,14 @@ EmberAfTvChannelInfo TvChannelManager::proxyChangeChannelRequest(std::string mat
 {
     // TODO: Insert code here
     EmberAfTvChannelInfo channel = {};
+    char affiliateCallSign[]     = "exampleResponseASign";
+    char callSign[]              = "exampleResponseCSign";
+    char name[]                  = "exampleResponseName";
+    channel.affiliateCallSign    = ByteSpan(Uint8::from_char(affiliateCallSign), sizeof(affiliateCallSign));
+    channel.callSign             = ByteSpan(Uint8::from_char(callSign), sizeof(callSign));
+    channel.name                 = ByteSpan(Uint8::from_char(name), sizeof(name));
+    channel.majorNumber          = static_cast<uint8_t>(1);
+    channel.minorNumber          = static_cast<uint16_t>(2);
     return channel;
```

Comment:
This is returning an object with pointers into this stack frame (e.g. pointing at `affiliateCallSign`).  That's not really ok....

Reply:
Removed the code as it's not needed at the moment.